PluginClasses/ServiceApi.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  ServiceApi.swift
+//  TVADataBinder
+//
+//  Created by MSApps on 16/04/2020.
```

Comment:
Please use the following enum when checking for http codes.
https://ibm-swift.github.io/Kitura-net/Enums/HTTPStatusCode.html